model/src/funTest/kotlin/utils/PostgresProvenanceFileStorageFunTest.kt

```diff
@@ -19,12 +19,10 @@
 
 package org.ossreviewtoolkit.model.utils
 
-import io.kotest.core.TestConfiguration
 import io.kotest.core.spec.style.WordSpec
-import io.kotest.engine.spec.tempfile
 import io.kotest.matchers.shouldBe
 
-import java.io.File
+import java.io.InputStream
```

Comment:
Please explain the benefits in the commit message.

Reply:
The benefits are implied by the linked issue #7118, but I can briefly repeat them in the commit message.